doc/news/_preparation_next_release.md

```diff
@@ -41,7 +41,42 @@ From now on `keyNew ("/array/#10", KEY_END)` will create a `Key` with name `/arr
 The whole internal implementation for `keySetName`, `keyAddName`, etc. has been completely rewritten.
 If you rely on specific behaviour of Elektra's Keynames and have already taken the two changes above into account, please refer to the newly created [Keyname documentation](../keynames) and (easier to read) [Python reference implementation](../keynames/keynames.py).
 
-<!-- TODO (kodebach): finish -->
+**Other keyname-related changes**
+
+- Metakeys now use the namespace `meta:/`.
+  The accessor function `keyGetMeta` and `keySetMeta` automatically add this namespace to preserve compatibility.
+  However, if you use the recently introduced `keyMeta` or otherwise directly access the keyname of a metakey, you will have to update your code.
+
+- There was an internal update to how warnings are generated.
```

Comment:
I would put this to a different section as it is only vaguely related to the key name change.

Reply:
I put it here, because I wanted to keep it at the top of the release notes. But creating a separate "Highlights" section just for this change seemed a bit extreme...

Reply:
Maybe to Compatibility?

Reply:
I will move it there. And maybe we should move the Compatibility section to the top (after Highlights)?

Reply:
Yes, good idea.

Reply:
I decided to instead create a "Breaking Changes" section _above_ Highlights with links to the full information.